code/game/mecha/mech_bay.dm

```diff
@@ -165,6 +197,75 @@
 	else
 		return 0
 
+/obj/machinery/mech_bay_recharge_port/update_icon()
+	. = ..()
+	icon_state = "[initial(icon_state)][anchored? "":"-unwrenched"]"
+
+/obj/machinery/mech_bay_recharge_port/wrenchAnchor()
+	. = ..()
+	update_icon()
+	if(!anchored)
+		stop_charge()
+		delink_devices()
+
+/obj/machinery/mech_bay_recharge_port/examine(mob/user)
+	. = ..()
+	if(recharge_floor)
+		var/direction = getDirString(recharge_floor)
+		to_chat(user,"<span class='notice'>Linked to \the [recharge_floor] at the [direction].</span>")
+	if(recharge_console)
+		var/direction = getDirString(recharge_console)
+		to_chat(user,"<span class='notice'>Linked to \the [recharge_console] at the [direction].</span>")
```

Comment:
though nice QoL, i feel like this feedback is kinda weird

Reply:
As in it's worded awkwardly?

Reply:
I'll retract this, because I now realize why you added it.